### ISSUE TITLE
get-devices reports "ERROR: Door '' is defined more than once in configuration_"

### ISSUE THREAD

**Issue by uhppoted** [uhppoted/uhppote-cli#5]
```
uhppote-cli --debug get-devices

... using default configuration from C:\ProgramData\uhppoted\uhppoted.conf
   ERROR: Door '' is defined more than once in configuration

My config file C:\ProgramData\uhppoted\uhppoted.conf
 
# SYSTEM
bind.address = 192.168.1.131:0
broadcast.address = 192.168.1.255:60000
listen.address = 192.168.1.131:60001

# REST
rest.http.enabled = true
rest.http.port = 8080
rest.https.enabled = false
rest.https.port = 8443
rest.CORS.enabled = true
rest.auth.enabled = false

# DEVICES
UT0311-L0x.303986753.address = 192.168.1.131:60000
UT0311-L0x.303986753.rollover = 100000
UT0311-L0x.303986753.door.1 = Great Hall

 ```

**Comment by twystd** [comment]
`uhppoted-api.Config` was updated in [commit c8c831f](https://github.com/uhppoted/uhppoted-api/commit/c8c831f905d7620e651e02588aa2dd40ce3da068) to handle _blank_ doors in the configuration file.

Have updated dependencies in all sub-projects to include this fix:
- [uhppote-cli](https://github.com/uhppoted/uhppote-cli)
- [uhppote-simulator](https://github.com/uhppoted/uhppote-simulator)
- [uhppoted-rest](https://github.com/uhppoted/uhppoted-rest)
- [uhppoted-mqtt](https://github.com/uhppoted/uhppoted-mqtt)
- [uhppoted-app-s3](https://github.com/uhppoted/uhppoted-app-s3)
- [uhppoted-app-sheets](https://github.com/uhppoted/uhppoted-app-sheets)
- [uhppoted-app-wild-apricot](https://github.com/uhppoted/uhppoted-app-wild-apricot)
- [uhppoted-httpd](https://github.com/uhppoted/uhppoted-httpd)

Leaving this issue open pending additional testing.

**Comment by twystd** [comment]
Fix verified - leaving this issue open pending release

**Comment by uhppoted** [comment]
Fixed in release v0.6.12
